test/data/OwnersSpec.scala

```diff
@@ -5,14 +5,6 @@ import org.specs2.mutable._
 
 class OwnersSpec extends Specification {
 
-  "A guardian stack" should {
-    "have only one owner" in {
```

Comment:
A stack having a single owner is a requirement. We didn't want shared ownership. If I remember correctly that's the main reason the `stacks` were stored as they were.
How is this enforced in this PR?